### DISCUSSION TITLE
What is the role of Application URL setting?

### DISCUSSION THREAD

**Discussion by einarpersson** [pocketbase/pocketbase#3699] [Q&A] [answered]
Context: I am configuring SMTP

Question(s): 
1. What is the role of the Application URL in settings _exactly_? I see that it is used in some default templates, but is it used in other places as well?
2. Should the Application URL point towards the Pocketbase instance or my main application (frontend)? I am running Sveltekit at `mydomain.com` and pocketbase on `mydomain.fly.dev`.

**Comment by ganigeorgiev** [comment, marked as answer]
Currently it is used only in the mail templates as a value for the `{APP_URL}` placeholder.
It can be used also by the users when sending programmatically custom emails (eg. `app.Settings().Meta.AppUrl`).
